### PR TITLE
Reduces lag quite significantly

### PR DESCRIPTION
**Pull request by Amelia0010**
### Intent of your Pull Request

reduces lag by about 99.99%
also reduces compile time by 99.99%

### CODE REVIEW

**Comment by Amelia0010** [comment]
image

**Comment by ThatLing** [comment]
Yogstation is strictly a high roleplay server and should be treated as such. We will not accept any joke PRs.

**Comment by Amelia0010** [comment]
this isn't a joke >:(

**Comment by Amelia0010** [comment]
compiles on local, haven't updated travis, hang on

**Comment by ghost** [comment]
Maps with diff:

<details><summary>lavaland_biodome_clown_planet</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>lavaland_surface_I</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>lavaland_surface_abductorterrorship</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>lavaland_surface_ash_walker1</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>lavaland_surface_clownfacility</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>lavaland_surface_cugganscove</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>lavaland_surface_golem_ship</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>lavaland_surface_goliathmound</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>lavaland_surface_hermit</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>lavaland_surface_legionbarracks</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>lavaland_surface_medical</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>lavaland_surface_mimingdrill</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>lavaland_surface_ntsurveyor</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>lavaland_surface_syndicatepod</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>lavaland_surface_weavernest</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>lavaland_surface_xenofacility</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>Academy</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>deepstorage</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>Cabin</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>beach</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>beach2</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>caves</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>challenge</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>moonoutpost19</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>research</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>snowdin</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>undergroundoutpost45</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>AsteroidStation</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>dreamstation04</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<details><summary>EfficiencyStation</summary>

Old | New | Status
--- | --- | ---
 |  | Deleted

</details>
<br>Last updated from merging commit 85eb0500a1c42a88e8e95e9465d7a86a58861b45 into d088176e898fe4a8f63093f15217541af1c7c4e5

File: code/modules/events/holiday/xmas.dm [hunk 1]
```diff
@@ -1,672 +0,0 @@
...
-		other_half.cracked = 1
-		other_half.icon_state = "cracker2"
-		target.put_in_active_hand(other_half)
-		playsound(user, 'sound/effects/snap.ogg', 50, 1)
-		return 1
-	return ..()
-
-/obj/item/clothing/head/festive
-	name = "festive paper hat"
-	icon_state = "xmashat"
```
Comment (ThatLing) [inline code/modules/events/holiday/xmas.dm:77]: Why are you removing christmas hats just before christmas?

Comment (Amelia0010) [reply code/modules/events/holiday/xmas.dm:77]: they caused about 0.0000001% of the lag, and I honestly can't say that christmas hats are worth that

**Comment by Ikoden** [comment]
Low effort PR, yet less buggy than what we are on at the moment.

**Comment by Amelia0010** [comment]
it's not low effort mate. this took me like, 20 minutes to make

**Comment by Jc1469** [comment]
stop buffing cult -1

**Comment by Cruix** [comment]
Good MapDiffBot test

**Pull request closed without merging**